<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>First</class>
 <widget class="QWidget" name="First">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="openFileButton">
       <property name="text">
        <string>Open file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="getStatistick">
       <property name="text">
        <string>Get statistic</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>RGB: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="redSpinBox">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="greenSpinBox">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="blueSpinBox">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="imageLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
